dwl.c

```diff
@@ -1791,12 +1792,18 @@ printstatus(void)
 				urg |= c->tags;
 		}
 		if ((c = focustop(m))) {
-			printf("%s title %s\n", m->wlr_output->name, client_get_title(c));
+			if (!(title = client_get_title(c)))
+				title = broken;
+			if (!(appid = client_get_appid(c)))
+				appid = broken;
+			printf("%s title %s\n", m->wlr_output->name, title);
+			printf("%s appid %s\n", m->wlr_output->name, appid);
```

Comment:
```suggestion
			title = client_get_title(c);
			appid = client_get_appid(c);
			printf("%s title %s\n", m->wlr_output->name, title ? title : broken);
			printf("%s appid %s\n", m->wlr_output->name, appid ? appid : broken);
```

Reply:
Looks good to me. Should I commit that edit?

Reply:
> Should I commit that edit?

Yes, please.